hrui/hrui.php

```diff
@@ -33,13 +33,19 @@ function hrui_civicrm_pageRun($page) {
   if ($page instanceof CRM_Contact_Page_DashBoard) {
     CRM_Utils_System::setTitle(ts('CiviHR Home'));
   }
+
   if ($page instanceof CRM_Contact_Page_View_Summary) {
     CRM_Core_Resources::singleton()
       ->addScriptFile('org.civicrm.hrui', 'js/contact.js')
       ->addScriptFile('org.civicrm.hrui', 'js/hrui.js')
-      ->addSetting(array('pageName' => 'viewSummary'));
+      ->addSetting(array('pageName' => 'viewSummary'))
+      ->addSetting(array('extensionsEnabled' => array(
+        'taskAssignments' => !!CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Extension', 'uk.co.compucorp.civicrm.tasksassignments', 'is_active', 'full_name')
+      )));
```

Comment:
you can wrap checking for T&A if enabled or not in a seperate method , something like what i did here for hrcase https://github.com/civicrm/civihr/blob/ab6a7b72984a481717f2535bbeaf90f0dce98ea1/hrcase/hrcase.php#L506

also it's better not to do my mistake and make the method receive any extension name ( for more usability in the feature )